src/switch_rtp.c

```diff
@@ -3473,8 +3473,13 @@ SWITCH_DECLARE(switch_status_t) switch_rtp_fork_set(switch_rtp_t *rtp_session, s
 		uint32_t ptime_in = 20, ptime_out = 20;
 		uint8_t pt = 0;
 
+		if ((strncmp(fork->rtp_codec, "pcma", 5)) && (strncmp(fork->rtp_codec, "pcmu", 5))) {
```

Comment:
Since fork->rtp_codec is copied from rtp_codec_iananame. Is it better to validate that when we get the value for rtp_codec_iananame?

Reply:
Same as above. rtp_codec_iananame needs to be lower cased, that is stored in fork->rtp_codec